drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -155,7 +154,7 @@ def characterize_gaia_distribution(hap_obj, json_timestamp=None, json_time_since
                           "mean neighbor distance",
                           "standard deviation of neighbor distances"]
     for item_value, item_title in zip(min_sep_stats, min_sep_title_list):
-        log.info("{} ({}): {}".format(item_title, out_dict["units"], item_value))
+        log.info("{} ({}): {}".format(item_title, "pixles", item_value))
```

Comment:
Correct spelling.

Reply:
Resolved in commit [af26058](https://github.com/spacetelescope/drizzlepac/pull/682/commits/af26058f4c31b40e1f1792634717a14ed8040aed)